src/main/java/frc/robot/Robot.java

```diff
@@ -67,7 +93,7 @@ public void autonomousPeriodic() {
 
     @Override
     public void autonomousExit() {
-
+    autoChooser.getSelected().cancel();
```

Comment:
this is not the greatest way to implement what I asked...

`getSelected` may have changed by the time autonomous has finished...

Instead, in `autonomousInit` call `getSelected` once. If it is `null` do nothing, otherwise, save it to a specialized member and schedule. 

In `autonomousExit`, if the member is not `null`, cancel it and nullify the member.